fdbmonitor/fdbmonitor.h

```diff
@@ -31,6 +31,7 @@
 #include <string>
 #include <sstream>
 #include <random>
+#include <unordered_map>
```

Comment:
This seems to be forgotten as well. See it is used later in the file:

https://github.com/apple/foundationdb/blob/bf59b5434fe6aea429935868508347b623b250c8/fdbmonitor/fdbmonitor.h#L86

And fails in another PR check:

```
In file included from /Users/ec2-user/foundationdb/fdbmonitor/fdbmonitor.cpp:32:
/Users/ec2-user/foundationdb/fdbmonitor/fdbmonitor.h:86:6: error: no template named 'unordered_map' in namespace 'std'; did you mean 'unordered_set'?
std::unordered_map<int, std::unordered_set<std::string>> set_watches(std::string path, int ifd);
~~~~~^~~~~~~~~~~~~
     unordered_set
/Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk/usr/include/c++/v1/unordered_set:411:28: note: 'unordered_set' declared here
class _LIBCPP_TEMPLATE_VIS unordered_set
                           ^
/Users/ec2-user/foundationdb/fdbmonitor/fdbmonitor.cpp:66:13: error: no template named 'unordered_map' in namespace 'std'; did you mean 'unordered_set'?
extern std::unordered_map<ProcessID, std::unique_ptr<Command>> id_command;
       ~~~~~^~~~~~~~~~~~~
            unordered_set
/Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk/usr/include/c++/v1/unordered_set:411:28: note: 'unordered_set' declared here
class _LIBCPP_TEMPLATE_VIS unordered_set
                           ^
/Users/ec2-user/foundationdb/fdbmonitor/fdbmonitor.cpp:67:13: error: no template named 'unordered_map' in namespace 'std'; did you mean 'unordered_set'?
extern std::unordered_map<pid_t, ProcessID> pid_id;
       ~~~~~^~~~~~~~~~~~~
            unordered_set
/Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk/usr/include/c++/v1/unordered_set:411:28: note: 'unordered_set' declared here
class _LIBCPP_TEMPLATE_VIS unordered_set
                           ^
/Users/ec2-user/foundationdb/fdbmonitor/fdbmonitor.cpp:68:13: error: no template named 'unordered_map' in namespace 'std'; did you mean 'unordered_set'?
extern std::unordered_map<ProcessID, pid_t> id_pid;
       ~~~~~^~~~~~~~~~~~~
```

https://github.com/apple/foundationdb/pull/11574#issuecomment-2299899553